llvm/test/tools/llvm-objcopy/ELF/srec-writer.test

```diff
@@ -1,36 +1,40 @@
+## Check for basic functionality using an input file with
+## various section types, adresses, data, and no segments.
+
```

Comment:
Nit: as the comment above is tightly-linked to the RUN lines immediately below, I'd get rid of the blank line.

Reply:
Done